### ISSUE TITLE
Ability to independintly change console output  tab font size

### ISSUE THREAD

**Issue by snufas** [microsoft/vscode#91304]
Issue Type: <b>Feature Request</b>

At a moment there is no setting to change the size of the output tab, the font is same as editor.fontSize. 

"editor.fontSize"
changes the 2 different components of the app it shouldn’t be that way , I think it should be managed independently with setting like “consolOutputWindow.fontSize”
because of this issue at a moment “problems” tab and others has differentt font sizes. is it not a better option to manage all tabs with something like “console.FontSize” 

I like editor font bigger because I am visually impaired, and I use 40 px for the editor but because of the output size is saime as editor font the output console size is way to big reason being that the window itself is smaller and thus it is not comfortable to read. 
I am sure because of the css stuf in this software devs can add this quickly enough

 I know that I should not put this in saime issue but 
“editorLineNumber.activeForeground” does not work for me. 
![image](https://user-images.githubusercontent.com/14276936/75181898-2394e600-5737-11ea-9934-98df23507dee.png)

VS Code version: Code 1.42.1 (c47d83b293181d9be64f27ff093689e8e7aed054, 2020-02-11T14:45:59.656Z)
OS version: Windows_NT x64 10.0.17763

**Comment by isidorn** [comment]
Thanks for filling this and this is a fair feature request.
This is not supported at the moment, and for now forwarding this to @sandy081 as it is a feature request against the output.

**Comment by sandy081** [comment]
Changing output font size separately is supported using Language specific settings. Check following

```json
"[Log]": { // Normal output
   "editor.fontSize" : 14
},
"[log]": { // log output
   "editor.fontSize" : 14
}
```

**Comment by isidorn** [comment]
@sandy081 oh yeah, forgot about that. Smart!

**Comment by snufas** [comment]
@sandy081 
I get your point how ever for each language does not sound like a  feasible  solution if you want this globaly.  
It is important  for visual  accessibility  so please upvote

**Comment by isidorn** [comment]
@snufas this is already supported. So that is the reason why we closed this issue. Can you just try putting in your `settings.json`

F1 > Open Settings (JSON) and put this in the file please

```
"[log]": {
   "editor.fontSize" : 14
}
```

**Comment by snufas** [comment]
@isidorn
Does not work for me that is the reason i replyed
i am happy to dooi't what ever whay it does work how ever it does not.  screenshot below should illustrate that. 
  
![image](https://user-images.githubusercontent.com/14276936/75442441-6503dc00-5957-11ea-8cdb-aa6989ca6e5b.png)

**Comment by isidorn** [comment]
Our mistake, please use the following (so with capital **L**)

```
"[Log]": {
   "editor.fontSize" : 14
}
```

This nicely works for me
@sandy081 this should be a bit more user friendly, currently it is quite hidden. It could at least be case insensitive

<img width="1147" alt="Screenshot 2020-02-27 at 14 23 56" src="https://user-images.githubusercontent.com/1926584/75449172-cd10ed00-596c-11ea-8493-0e3824d92cc2.png">

**Comment by sandy081** [comment]
Yeah, I am aware if it. Currently `log` is used for log files and `Log` is used for normal output channels. This is because of some history. We have been using `Log` for output channels and we cannot change it. Log files are using `log` as language.

**Comment by isidorn** [comment]
Oh my 🙀

**Comment by snufas** [comment]
@sandy081 
![image](https://user-images.githubusercontent.com/14276936/75472671-c34ab200-598b-11ea-9268-5c718df9a265.png)

saime issue...

**Comment by isidorn** [comment]
@snufas in your picture the output is empty. Thus I would choose an output channel with lots of output for example GIT, and make sure to increase the fontSize. 14 is small, try 20.
I am certain this works.

**Comment by snufas** [comment]
@isidorn 
Sorry i don't want to contradict  with what you are saying how ever there it gos 
![image](https://user-images.githubusercontent.com/14276936/75491785-68768200-59ae-11ea-94ee-59dbc157c6d9.png)

i didin't put anything because the  cursor size alone should indicate that it is not  small enough , The text indicates that it is as big as my editor font .

**Comment by isidorn** [comment]
@snufas contraditcting me is perfectly fine :) For some reason this works fine for me (picture attached)
@sandy081 do you have any ideas what might be wrong here?

<img width="1004" alt="Screenshot 2020-02-28 at 11 21 31" src="https://user-images.githubusercontent.com/1926584/75540536-84b10800-5a1c-11ea-955d-31cdfcc1519f.png">

**Comment by sandy081** [comment]
@snufas May I know what output channel you are looking at? Can you also please try with `git` output channel?

**Comment by snufas** [comment]
@sandy081 
i use the default vs code terminal, I tryed with externals cmd and git saime  thing...

**Comment by sandy081** [comment]
I am now getting short of ideas. It is working for me. I am really not sure what is going wrong.

Can you please try with our insiders, that's one of the differences.
